.circleci/config.yml

```diff
@@ -26,9 +26,6 @@ jobs:
       - slack/notify:
           event: fail
           template: basic_fail_1
-      - slack/notify:
-          event: pass
-          template: success_tagged_deploy_1
```

Comment:
Is it going to send a notification on every success? Is it possible to make is send only if previous build has failed?

Reply:
I'll check the slack orb, but I'll create a separate PR for that. This is to close the current ticket.
Is that ok?